### PR TITLE
Remove unlockable content

### PR DESCRIPTION
**Pull request by antho1404**
Remove unlockable content from the core/react/doc

### CODE REVIEW

File: packages/core/src/uploader/index.ts [hunk 1]
```diff
@@ -7,29 +7,13 @@ export class Uploader {
...
-    isPrivate = false,
-  ) {
+  public async upload(file: File | URL | string | undefined | null) {
     if (file === null) return null
     if (!file) return
     if (file instanceof URL) return file.toString()
     if (typeof file === 'string') return file
     const formData = new FormData()
     formData.append('file', file)
-    formData.append('protected', isPrivate ? 'true' : 'false')
```
Comment (antho1404) [inline packages/core/src/uploader/index.ts:32]: @NicolasMahe could you confirm that removing this param will not have any unwanted side effects

Comment (NicolasMahe) [reply packages/core/src/uploader/index.ts:32]: All good. This `protected` param is optional.

File: docs/pages/tools/sdk-react/useCreateNFT.md [hunk 2]
```diff
@@ -36,7 +36,6 @@ export default function Component() {
             content: azukiImage, // content file uploaded by the user
             preview: azukiImagePreview, // preview in the case of private or animation content uploaded by user
```
Comment (NicolasMahe) [inline docs/pages/tools/sdk-react/useCreateNFT.md]: Preview should also be removed
```suggestion
            preview: azukiImagePreview, // preview in the case of animation content
```

File: docs/pages/tools/sdk-react/useCreateNFT.md [hunk 3]
```diff
@@ -125,7 +123,6 @@ Arguments:
       content: File           // Content for the asset
       preview?: File,         // Preview of the asset (if the content is private or an animation)
```
Comment (NicolasMahe) [inline docs/pages/tools/sdk-react/useCreateNFT.md]:
```suggestion
      preview?: File,         // Preview of the asset (if the content is an animation)
```

**Review by NicolasMahe — APPROVED** [review]
Good to merge after the change on two comments

File: docs/pages/tools/sdk-react/useCreateNFT.md (hunk 2 above)
Comment (antho1404) [reply docs/pages/tools/sdk-react/useCreateNFT.md]: preview is still useful if we set an animation

Comment (NicolasMahe) [reply docs/pages/tools/sdk-react/useCreateNFT.md]: Yeah I notice this after posting this first suggestion but forgot to change the message!

**Pull request merged**